docs/framework/react/devtools.md

```diff
@@ -13,14 +13,16 @@ When you begin your React Query journey, you'll want these devtools by your side
 
 ## Install and Import the Devtools
 
-The devtools are a separate package that you need to install:
+The devtools are a separate package that you need to install.
+
+The major version of all the packages must be in sync.
```

Comment:
@TkDodo 

Should we add this line to the `main` branch too?

Reply:
sure